### ISSUE TITLE
9.4.3 Duplicate Recurring Tickets

### ISSUE THREAD

**Issue by mauriciolamego** [glpi-project/glpi#6205]
**Describe the bug**

Recurring tickets are being opened in duplicate as the renewal date (info tab), based on frequency, is not being updated after the last event.
It is also noted that selecting a periodicity in “days or month” causes the GLPI to crash and therefore the action not to be taken. It is only accepting the periodicity in hours or 1 (one) day, but nevertheless occurs the duplicity of the call previously reported.

**Page(s) URL**

https://GLPI/front/ticketrecurrent.form.php?id=XX

**To reproduce**

 

1.	Go to '...'
ASSISTANCE> RECURRENT TICKET - Click on an existing recurring ticket.
The item in question is the one highlighted below.

![image](https://user-images.githubusercontent.com/40633272/62639326-aed29980-b915-11e9-80a3-cdbdd6939f9a.png)

After choosing the periodicity, click on the "information" tab.
If you choose up to 1 (one) day or hours you will not have any problems, unless the call will be opened in duplicate.
Duplicate call generation seems to be occurring due to the "information" tab not recognizing that the scheduled call has already been generated and now has to reschedule for the next period.
Choosing a longer period, for example 1 month, will lock the GLPI.
The error is: Duplicate calls.

![image](https://user-images.githubusercontent.com/40633272/62639445-e6d9dc80-b915-11e9-9ac3-e48c80b0987e.png)

**Expected behavior**

After the call is created in the specified period, it will only be registered in the next schedule, in the specified period.

**Your GLPI setup (you can find it in Setup > General menu, System tab)**

```

Information   about system installation and configuration
--
[code]       GLPI 9.4.3 (   => /home/storage/c/df/d8/GLPI/public_html)   Installation   mode: TARBALL
Server
Operating system:   Linux hm2501 4.4.165-grsec-1.x86_64 #1 SMP Wed Nov 28 21:23:35 UTC 2018   x86_64   PHP 7.1.13   cgi-fcgi (Core, PDO, PDO_Firebird, PDO_ODBC, Phar, Reflection, SPL,   SimpleXML, bcmath, bz2, calendar, cgi-fcgi,           ctype, curl, date, dba, dom, enchant,   exif, fileinfo, filter, ftp, gd, gettext, gmp, hash, iconv, imap, interbase,   intl, json,           ldap, libxml, mbstring, mcrypt,   mysqli, odbc, openssl, pcntl, pcre, pdo_dblib, pdo_mysql, pdo_pgsql,   pdo_sqlite, pgsql, posix,           pspell, readline, recode, session,   shmop, snmp, soap, sockets, standard, sysvmsg, sysvsem, sysvshm, tidy,   timezonedb, tokenizer,           wddx, xml, xmlreader, xmlrpc,   xmlwriter, xsl, zip, zlib)   Setup:   max_execution_time="60" memory_limit="128M"   post_max_size="50M" safe_mode=""   session.save_handler="files"           upload_max_filesize="50M"    Software: Apache   ()           Mozilla/5.0 (Windows NT 10.0; WOW64)   AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.75 Safari/537.36   Server Software:   Percona Server (GPL), Release 81.0, Revision c96c427           Server Version: 5.6.35-81.0-log           Server SQL Mode:            Parameters: glpi_p@glpi_p.mysql.dbaas.com.br/glpi_p           Host info: glpi_p.mysql.dbaas.com.br   via TCP/IP              mysqli extension   is installed   ctype extension   is installed   fileinfo   extension is installed   json extension is   installed   mbstring   extension is installed   iconv extension   is installed   zlib extension is   installed   curl extension is   installed   gd extension is   installed   simplexml   extension is installed   xml extension is   installed   ldap extension is   installed   imap extension is   installed   xmlrpc extension   is installed   exif extension is   installed   Zend OPcache extension is not   present   APCu extension is not present   CAS extension is not present   Database version   seems correct (5.6.35) - Perfect!   /home/storage/c/df/d8/GLPI/public_html/files/_log   : OK   /home/storage/c/df/d8/GLPI/public_html/config   : OK   /home/storage/c/df/d8/GLPI/public_html/files   : OK   /home/storage/c/df/d8/GLPI/public_html/files/_dumps   : OK   /home/storage/c/df/d8/GLPI/public_html/files/_sessions   : OK   /home/storage/c/df/d8/GLPI/public_html/files/_cron   : OK   /home/storage/c/df/d8/GLPI/public_html/files/_graphs   : OK   /home/storage/c/df/d8/GLPI/public_html/files/_lock   : OK   /home/storage/c/df/d8/GLPI/public_html/files/_plugins   : OK   /home/storage/c/df/d8/GLPI/public_html/files/_tmp   : OK   /home/storage/c/df/d8/GLPI/public_html/files/_cache   : OK   /home/storage/c/df/d8/GLPI/public_html/files/_rss   : OK   /home/storage/c/df/d8/GLPI/public_html/files/_uploads   : OK   /home/storage/c/df/d8/GLPI/public_html/files/_pictures   : OK   Web access to the files directory   should not be allowed but this cannot be checked automatically on this   instance.   Make sure acces   to error log file is   forbidden; otherwise review .htaccess file and web server configuration.SELinux mode is Disabled
Libraries
htmLawed version   1.2.4 in (/home/storage/c/df/d8/GLPI/public_html/lib/htmlawed)   phpmailer/phpmailer   version 6.0.7 in (/home/storage/c/df/d8/GLPI/public_html/vendor/phpmailer/phpmailer/src)   simplepie/simplepie   version 1.5.2 in (/home/storage/c/df/d8/GLPI/public_html/vendor/simplepie/simplepie/library)   tecnickcom/tcpdf   version 6.2.26 in (/home/storage/c/df/d8/GLPI/public_html/vendor/tecnickcom/tcpdf)   michelf/php-markdown   in (/home/storage/c/df/d8/GLPI/public_html/vendor/michelf/php-markdown/Michelf)   true/punycode in   (/home/storage/c/df/d8/GLPI/public_html/vendor/true/punycode/src)   iamcal/lib_autolink   in (/home/storage/c/df/d8/GLPI/public_html/vendor/iamcal/lib_autolink)   sabre/vobject in   (/home/storage/c/df/d8/GLPI/public_html/vendor/sabre/vobject/lib)   zendframework/zend-cache   in (/home/storage/c/df/d8/GLPI/public_html/vendor/zendframework/zend-cache/src)   zendframework/zend-i18n   in (/home/storage/c/df/d8/GLPI/public_html/vendor/zendframework/zend-i18n/src)   zendframework/zend-serializer   in (/home/storage/c/df/d8/GLPI/public_html/vendor/zendframework/zend-serializer/src)   monolog/monolog   in (/home/storage/c/df/d8/GLPI/public_html/vendor/monolog/monolog/src/Monolog)   sebastian/diff in   (/home/storage/c/df/d8/GLPI/public_html/vendor/sebastian/diff/src)   elvanto/litemoji   in (/home/storage/c/df/d8/GLPI/public_html/vendor/elvanto/litemoji/src)   symfony/console   in (/home/storage/c/df/d8/GLPI/public_html/vendor/symfony/console)   leafo/scssphp in   (/home/storage/c/df/d8/GLPI/public_html/vendor/leafo/scssphp/src)
SQL   replicas
Not active
Notifications
Way of sending   emails: SMTP+TLS (atendimento@solemtic.com.br@email-ssl.com.br)
Mails   receivers
Name: 'destinatario@dominio.com'   Active: Yes           Server:   '{email-ssl.com.br:993/imap/ssl}INBOX' Login: 'Destinatario@dominio.com'   Password: Yes
Plugins   list
formcreator          Name: Form Creator                   Version: 2.8.3      State: Enabled           fusioninventory      Name: FusionInventory                Version: 9.4+1.1    State: Enabled
[/code]
To   copy/paste in your support request


```

**Additional context**

I advise that even running the proposed solutions in cases # 6159 or # 6140 the problem is not solved.

**Comment by chauchse** [comment]
I have the same problem

**Comment by flegastelois** [comment]
Hi,

Thanks for your report.

I confirm, this happens when we choose a begin date outside the opening hours of the selected calendar.

We'll fix it in a future release, if you'd like to help us produce a fix faster: subscribe to GLPI Network at https://services.glpi-network.com

Kind regards,

**Comment by cedric-anne** [comment]
Hi @mauriciolamego @chauchse ,

Could you try the fix from #6275 ?

Regards
